x-pack/plugins/cloud_security_posture/public/components/cloud_posture_page.test.tsx

```diff
@@ -27,7 +27,7 @@ import { useLicenseManagementLocatorApi } from '../common/api/use_license_manage
 
 const chance = new Chance();
 
-jest.mock('../common/api/use_setup_status_api');
+jest.mock('@kbn/cloud-security-posture'); // Mock use_setup_status_api
```

Comment:
you are mocking the whole package, the comment doesn't really help here. I think there should be a way to mock a specific import from a package